ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml

```diff
@@ -15,7 +15,11 @@ presubmits:
         image: registry.svc.ci.openshift.org/ci/promrules:latest
         imagePullPolicy: Always
         name: ""
-        resources: {}
+        resources:
+          limits:
+            cpu: 500m
```

Comment:
We're certain of these values for jobs that run ci-op, but I'm not absolutely certain about these jobs running other code. Once we change this if we see jobs goiing over the limit we can adjust

Reply:
Most of these seem to be small utilities, I think the limits should be fine.